tardis/tests/tests_slow/conftest.py

```diff
@@ -66,8 +78,21 @@ def data_path(request):
     }
 
 
+@pytest.fixture(scope="session")
+def gen_ref_dirpath(request):
+    if request.config.getoption("--generate-reference"):
+        integration_tests_config = request.config.option.integration_tests_config
+        path = os.path.join(os.path.expandvars(os.path.expanduser(
+            integration_tests_config["generate_reference"])), tardis_githash
```

Comment:
@wkerzendorf @yeganer How about naming the directory as shortened githash ( `tardis_githash[:7]` ) ?